libraries/MySensors/MySensor.cpp

```diff
@@ -18,8 +18,29 @@
  */
 
 
+
 #include "MySensor.h"
 
+
```

Comment:
Below is where the "magic" happens. The .cpp includes will be optioned by #defines created in sketch...